source/utilities.tex

```diff
@@ -15010,13 +15011,11 @@
 \begin{itemize}
 \item \oldconcept{DefaultConstructible} (\tref{cpp17.defaultconstructible}),
 \item \oldconcept{CopyConstructible} (\tref{cpp17.copyconstructible}),
-\item \oldconcept{CopyAssignable} (\tref{cpp17.copyassignable}), and
+\item \oldconcept{CopyAssignable} (\tref{cpp17.copyassignable}),
+\item \oldconcept{Swappable}\iref{swappable.requirements}, and
 \item \oldconcept{Destructible} (\tref{cpp17.destructible})
 \end{itemize}
-requirements,
-
-\item
-it is swappable\iref{swappable.requirements} for lvalues, and
+requirements, and
```

Comment:
I think it's debatable whether we need to use an itemized list now that we only have two top-level items. Maybe it's a bit more readable to have each requirement on its own line like that, but it doesn't match any of the other lists of *Cpp17Foo* requirements being altered by this change.

Reply:
I'll send a separate PR.